app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1><%= t("transactions.new.create") %></h1>
+
+<%= render "books/book_card" %>
+
+<%= form_with(model: [@book, @transaction], local: true) do |form| %>
+  <%= render "shared/object_error_messages", object: form.object %>
+
+  <%= form.hidden_field :user_id, value: current_user&.id %>
```

Comment:
theo chị hiểu em đang thiết kế phải login mới vào dc page này, vậy thfi cần toán tử an toàn ko
chị thấy cần bổ sung action filter ở controller là dc